docs/usage/shoot-operations/shoot_updates.md

```diff
@@ -53,21 +53,44 @@ We consider shoot upgrades to change either the:
 Generally, an upgrade is also performed through a reconciliation of the `Shoot` resource, i.e., the same concepts as for [shoot updates](#updates) apply.
 If an end-user triggers an upgrade (e.g., by changing the Kubernetes version) after a new Gardener version was deployed but before the shoot was reconciled again, then this upgrade might incorporate the changes delivered with this new Gardener version.
 
+The `UpdateStrategy` field in the Shoot specification (`.spec.provider.workers[].updateStrategy`) gives users the flexibility to define how the `machine-controller-manager`(MCM) handles worker pool updates during the upgrade process. Currently gardener support three update strategies:
+* `AutoRollingUpdate`
+* `AutoInPlaceUpdate`
+* `ManualInPlaceUpdate`
+
 ### In-Place vs. Rolling Updates
 
-If the Kubernetes patch version is changed, then the upgrade happens in-place.
-This means that the shoot worker nodes remain untouched and only the `kubelet` process restarts with the new Kubernetes version binary.
-The same applies for configuration changes of the kubelet.
+#### In-Place Updates
+- The upgrade is performed in a "rolling update" manner, without replacing the underlying machines.
+- The shoot worker nodes remain intact and are not recreated.
+- The existing workload is gracefully drained and evicted from the worker nodes, respecting the configured `PodDisruptionBudget`s (see [Specifying a Disruption Budget for your Application](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)).
+- **No Draining Scenarios**:
+  - When the Kubernetes patch version is updated, the upgrade is executed without draining the node. The shoot worker nodes remain unchanged, and only the `kubelet` process is restarted with the updated Kubernetes version binary.
```

Comment:
This is also the case for Rolling update strategies, putting it under this heading might be confusing